src/searchModal/scss/advancedSearch.scss

```diff
@@ -21,6 +21,13 @@ $invalidCriteriaBorder: #266d9c;
             top: 2px;
             margin-right: 8px;
         }
+        .icon-loop {
+            font-size: 1.6rem;
+            position: relative;
+            top: 2px;
+            margin-right: 8px;
+            animation: rotating 2s linear infinite;
+        }
```

Comment:
This is not animated due to inline display of a span. You can add `display: inline-block;` to fix the issue

Reply:
Done